[AArch64] Extend v2i64 fptosi.sat to v2f64

davemgreen opened the pull request:
This helps it produce a single instruction for the saturate, as opposed to having to scalarize.

artagnon reviewed (commented):
This looks good to me, but please wait for one other reviewer.

efriedma-quic reviewed (approved):
LGTM

davemgreen committed:
[AArch64] Extend v2i64 fptosi.sat to v2f64

This helps it produce a single instruction for the saturate, as opposed to
having to scalarize.
